SgmlTests/SgmlTests.csproj

```diff
@@ -7,15 +7,8 @@
     <RepositoryUrl>https://github.com/lovettchris/SgmlReader/</RepositoryUrl>
   </PropertyGroup>
 
-  <ItemGroup>
-    <None Remove="Resources\62.test" />
-  </ItemGroup>
```

Comment:
Your earlier commit (prior to my changes) originally added this, I assumed it wasn't intentional so I removed it.

Reply:
Thanks, might help to also sort the last 3 EmbeddedResource entries in this file...